src/Kros.AspNetCore/Extensions/HttpClientExtensions.cs

```diff
@@ -42,6 +44,45 @@ public static class HttpClientExtensions
             return response;
         }
 
+        /// <summary>
+        /// Get response from http request (with a body) and throws exception if request is unsuccessful.
+        /// </summary>
+        /// <param name="client">Http client.</param>
+        /// <param name="uri">Uri for request.</param>
+        /// <param name="body">Request body.</param>
+        /// <param name="defaultException">Default exception to by thrown, if failure result code is unsupported.</param>
+        public static async Task<HttpResponseMessage> GetAndCheckResponseAsync(
+            this HttpClient client,
+            Uri uri,
+            object body,
+            Exception defaultException = null)
+        {
+            string jsonBody = JsonSerializer.Serialize(body);
+            var request = new HttpRequestMessage
+            {
+                Method = HttpMethod.Get,
+                RequestUri = uri,
+                Content = new StringContent(jsonBody, Encoding.UTF8, "application/json")
+            };
+            HttpResponseMessage response = await client.SendAsync(request);
+            response.ThrowIfNotSuccessStatusCode(defaultException);
+            return response;
+        }
+
+        /// <summary>
+        /// Get response from http request (with a body) and throws exception if request is unsuccessful.
+        /// </summary>
+        /// <param name="client">Http client.</param>
+        /// <param name="url">Url for request.</param>
+        /// <param name="body">Request body.</param>
+        /// <param name="defaultException">Default exception to by thrown, if failure result code is unsupported.</param>
```

Comment:
Je to `Ctrl + C, Ctrl + V`, ale máme tam chybu: _Default exception to **be** thrown…_ Oprav to pls. všade.